tools/benchmark/README.md

```diff
@@ -2,8 +2,12 @@
 
 `etcd/tools/benchmark` is the official benchmarking tool for etcd clusters.
 
-## Install
-
-```bash
-go get github.com/coreos/etcd/tools/benchmark
+## Download and install
+To get `benchmark` from the `master` branch via `go get`:
+```sh
+$ go get github.com/coreos/etcd/tools/benchmark
```

Comment:
Sorry I forgot to mention in my previous comment, can we also update the link to: 
go get go.etcd.io/etcd/tools/benchmark
The etcd repo was moved to go.etcd.io (see #9965).  Both links work because github is redirecting the old url to the new one. But I think it is better we update the link to reflect the new repo name.

Reply:
Thanks, get it.